cli/src/main/resources/soapenvelope12_xmlprotocol.scala.template

```diff
@@ -6,7 +6,7 @@ usage:
 val obj = scalaxb.fromXML[soapenvelope12.Foo](node)
 val document = scalaxb.toXML[soapenvelope12.Foo](obj, "foo", soapenvelope12.defaultScope)
 **/
-object `package` extends XMLProtocol { }
+object XMLProtocol extends XMLProtocol
```

Comment:
This is going to be a huge change for anyone who's currently using scalaxb, so I won't do this unless we really have to.